### PR TITLE
Add csp urls for ads

### PR DESCRIPTION
**Pull request by thekp**
A part of: #6774

**Overall change:**
_Add links to our CSP header required for the current Ads._

example url: to see locally: http://localhost:7080/mundo?adUnit=/4817/bbccom.test.site.flash/news_homepage_int

To see CSP headers being injected locally, you will have to update this line: https://github.com/bbc/simorgh/blob/b04a7263a1422ccb060392caf05999a086dd3c2e/src/server/index.jsx#L100

to return `injectCspHeader` when true, i.e:

```
const injectCspHeaderProdBuild =
  process.env.NODE_ENV !== 'production' ? injectCspHeader : injectCspHeader;
```

**Code changes:**

_list of new urls added to our csp header for nonLiveCanonicalPages:_

connect-src: 
```
      'https://securepubads.g.doubleclick.net',
      'https://pagead2.googlesyndication.com',
      'https://survey.effectivemeasure.net',
      'https://www.bbc.co.uk',
      'https://static.test.files.bbci.co.uk',
      'https://experience.tinypass.com',
      'https://detect-survey.effectivemeasure.net',
      'https://collector.effectivemeasure.net/',
      'https://csi.gstatic.com',
```

frame-src:
```
      'https://tpc.googlesyndication.com',
      'https://bcp.crwdcntrl.net',
      'https://edigitalsurvey.com',
      'https://*.safeframe.googlesyndication.com/',
```

image-src:
```
      'data:',
      'https://pagead2.googlesyndication.com',
      'https://tpc.googlesyndication.com',
      'https://securepubads.g.doubleclick.net',
      'https://www.google.com',
      'https://secure-us.imrworldwide.com',
      'https://collector.effectivemeasure.net',
      'https://sb.scorecardresearch.com',
```

script-src:
```
      'https://tpc.googlesyndication.com',
      'https://privacy.crwdcntrl.net',
      'https://gn-web-assets.api.bbc.com',
      'https://securepubads.g.doubleclick.net',
      'https://bcp.crwdcntrl.net',
      'https://ad.crwdcntrl.net',
      'https://sb.scorecardresearch.com',
      'https://me-ssl.effectivemeasure.net',
      'https://bbc.gscontxt.net',
      'https://adservice.google.co.uk',
      'https://adservice.google.com',
      'https://t.effectivemeasure.net',
      'https://tags.crwdcntrl.net',
      'https://collector.effectivemeasure.net',
      'https://cdn.ampproject.org',
      "'unsafe-eval'",
```

default-src
```
      'https://*.safeframe.googlesyndication.com',
      'https://tpc.googlesyndication.com',
```

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Comment by thekp** [comment]
Note: 

image


`'http://localhost:1124/sockjs-node/info?t=1592227698658'` csp header complaint comes from hot module reload, if you run this pr with `npm run build && npm run start` it should not appear in the console.

**Comment by mmcculloch** [comment]
> Note:
> 
> image
> 
> `'http://localhost:1124/sockjs-node/info?t=1592227698658'` csp header complaint comes from hot module reload, if you run this pr with `npm run build && npm run start` it should not appear in the console.

It also only happens if you enable CSP locally so shouldn't happen otherwise.

**Review by mmcculloch — APPROVED** [review]

**Review by DenisHdz — COMMENTED** [review]
Tested it using this Chrome [extension](https://chrome.google.com/webstore/detail/csp-evaluator/fjohamlofnakbnbfjkohkbdigoodcejf?hl=en) and all the directives look fine except for 
Screenshot 2020-06-15 at 16 41 31
Screenshot 2020-06-15 at 16 43 58

We are aware of the `unsafe-inline` and there is a [PR](https://github.com/bbc/simorgh/pull/6838) in progress to remove it, however, we should find out about the `https://securepubads.g.doubleclick.net`.

File: src/server/utilities/cspHeader/index.js [hunk 1]
```diff
@@ -270,8 +297,26 @@ const directives = {
...
+      'https://adservice.google.co.uk',
+      'https://adservice.google.com',
+      'https://t.effectivemeasure.net',
+      'https://tags.crwdcntrl.net',
+      'https://collector.effectivemeasure.net',
+      'https://cdn.ampproject.org',
+      // end dotcom required urls
       "'self'",
       "'unsafe-inline'",
+      "'unsafe-eval'",
```
Comment (tochwill) [inline src/server/utilities/cspHeader/index.js:319]: I'd be very wary of putting `unsafe-eval` in the live CSP, and should be one of the issues that we raise with GNL.

**Review by tochwill — APPROVED** [review]

**Pull request merged**